proto/astria/execution/v1alpha2/execution.proto

```diff
@@ -4,6 +4,17 @@ package astria.execution.v1alpha2;
 
 import "google/protobuf/timestamp.proto";
 
+// GenesisInfo contains the information needed to start a rollup chain.
+//
+// This information is used to determine which sequencer & celestia data to
+// use from the Astria & Celestia networks.
+message GenesisInfo {
+  bytes rollup_id = 1;
+  uint32 sequencer_genesis_block_number = 2;
+  uint32 celestia_base_block_number = 3;
```

Comment:
In cometbft these are called the heights, not numbers. I'd prefer their convention.